### PR TITLE
Fix transaction pool rejection

### PR DESCRIPTION
**Pull request by arkpar**
Looks like the code is missing a `return` here.

### CODE REVIEW

**Review by bkchr — APPROVED** [review]

**Comment by bkchr** [comment]
bot merge

File: client/service/src/lib.rs [hunk 1]
```diff
@@ -498,7 +498,7 @@ where
 	fn import(&self, transaction: B::Extrinsic) -> TransactionImportFuture {
 		if !self.imports_external_transactions {
 			debug!("Transaction rejected");
-			Box::pin(futures::future::ready(TransactionImport::None));
+			return Box::pin(futures::future::ready(TransactionImport::None));
```
Comment (bkchr) [inline client/service/src/lib.rs:501]:
```suggestion
			return Box::pin(futures::future::ready(TransactionImport::None))
```

**Comment by bkchr** [comment]
bot merge

**Review by tomusdrw — APPROVED** [review]
test would be good

**Pull request merged**